tracing_test.go

```diff
@@ -60,23 +65,47 @@ func TestTracingSpanAttributes(t *testing.T) {
 		DisableRelay:   true,
 	}
 	WithVerifiedServer(t, opts, func(ch *Channel, hostPort string) {
+		const (
+			customAppHeaderKey           = "futurama"
+			customAppHeaderExpectedValue = "simpsons"
+		)
+		var customAppHeaderValue atomic.Value
```

Comment:
we use `uber-go/atomic` in tchannel, which has a `atomic.String` that you could use here